### ISSUE TITLE
Seeking new maintainer

### ISSUE THREAD

**Issue by manub** [manub/scalatest-embedded-kafka#158]
I've realised I don't have enough time to dedicate to this project anymore, and I was wondering if somebody would be interested to help out with this. I know there's a few users out there so it would be great to be able to give them a better support.

If anybody is interested, just comment on this PR.

**Comment by NeQuissimus** [comment]
I am using this library quite regularly. However, I have no knowledge of the Confluent parts of this (Schema registry in particular).
I can definitely help out but I would want to split the Confluent parts off into a sub-module and have somebody else help out there

**Comment by francescopellegrini** [comment]
> I am using this library quite regularly. However, I have no knowledge of the Confluent parts of this (Schema registry in particular).
> I can definitely help out but I would want to split the Confluent parts off into a sub-module and have somebody else help out there

I may provide support on the Schema Registry module. =)

**Comment by manub** [comment]
One solution would be to give you both commit rights. We'd have to handle the release process though - although if requested I can still run releases. Probably also worth to move it to a new org?

**Comment by NeQuissimus** [comment]
Maybe rather than moving this repo, we should fork and archive this repo with a notice?

Then we could rename, break the compatibility (split modules) etc. ?
Just throwing it out there...

I can host you guys in my fork, or we create a single-repo org?!

**Comment by manub** [comment]
@NeQuissimus agreed with the approach. Better to create a single-repo org in my opinion.

**Comment by manub** [comment]
Created a new org and invited both of you. For now as members, we can discuss further there.

I suggest we close this ticket and we create another one for the migration. `2.1.0` is a good candidate to be released on the new org? (something like `"io.github.scalatestembeddedkafka"`)

**Comment by francescopellegrini** [comment]
Can we close this now?
